vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -798,7 +798,31 @@
       application.rolesToGroups:
         {{ roles_to_groups_yaml | to_nice_yaml | indent(2) }}
   when: arkcase_version == "" or arkcase_version is version('3.3.3', '>=')
-    
+
+- name: Make sure AWS credentials are insterted in the arkcase-server.yaml file if they are defined
+  become: yes
+  become_user: arkcase
+  lineinfile:
+    path: /opt/arkcase/data/arkcase-home/.arkcase/acm/acm-config-server-repo/arkcase-server.yaml
+    state: present
+    backup: True
+    backrefs: True
+    regexp: '^awsAccessKeyId'
```

Comment:
Typically the fact names are formatted as `aws_access_key_id` and they don't end in `fact`.